### ISSUE TITLE
Unable to compile widget attribut with model parameter

### ISSUE THREAD

**Issue by sanpii** [antoyo/relm#162]
Hello,

I try to compile relativly old code with rust 1.34.0-nightly. I have a compilation error for widget with model parameter in `view!` macro:

```
error: expected `,`                        
   --> src/edit.rs:199:60                                                          
    |                                        
199 |                           Calendar("Defer until".to_owned()) {           
    |  ____________________________________________________________^
200 | |                             CalendarUpdated(date) => Msg::UpdateDate(DateType::Threshold, date),
201 | |                         },                               
    | |_________________________^                                                  
```

[code](https://gitlab.com/sanpi/effitask/blob/59f9208ac7712522672f6dd77cd05a5e2cfbbbca/src/edit.rs#L199) — [compilation log](https://gitlab.com/sanpi/effitask/-/jobs/160222689)

**Comment by sanpii** [comment]
Here is a patch to reproduce the bug with tests:

```diff
diff --git i/tests/model-param-attribute.rs w/tests/model-param-attribute.rs
index 390089e..e44ded8 100644
--- i/tests/model-param-attribute.rs
+++ w/tests/model-param-attribute.rs
@@ -102,6 +102,8 @@ impl Widget for Win {
                 // Set the orientation property of the Box.
                 orientation: Vertical,
                 Button(self.model.initial_text),
+                Button(self.model.initial_text) {
+                },
                 #[name="label"]
                 gtk::Label {
                     // Bind the text property of the label to the counter attribute of the model.
```

**Comment by antoyo** [comment]
@sanpii Thanks. I'll fix that during the next Rust+GNOME Hackfest (should be in april).
I'll make sure to test that `mg` and `titanium` compile fine with the new parser in order to make sure that this does not happen anymore.
Sorry for the inconvenience.

**Comment by BlockCat** [comment]
Is there are temporary workaround for this issue?

**Comment by antoyo** [comment]
@BlockCat For now, you can either use an older version of relm, or create the widget without the attribute.

**Comment by antoyo** [comment]
Fixed by #167.
